docs/book/src/reference/rescaffold.md

```diff
@@ -29,7 +29,7 @@ Currently, it supports two optional params, `input-dir` and `output-dir`.
 
 `input-dir` is the path to the existing project that you want to re-scaffold. Default is the current working directory.
 
-`output-dir` is the path to the directory where you want to generate the new project. Default is a subdirectory in the current working directory.
+`output-dir` is the path to the directory where you want to generate the new project. Default is a directory named "output-dir" in the current working directory.
```

Comment:
If we not inform the `output-dir` it should be == $pwd , the current directory.
Are you facing a different behaviour?
Does it create a directory called `output-dir` and scaffold the project inside of it?

Reply:
So that's the part that confused me.

The flag itself defaults to "" -> pkg/cli/alpha/command.go:59

The struct itself has a default then later here -> pkg/cli/alpha/internal/generate.go:46

<img width="573" alt="image" src="https://github.com/user-attachments/assets/8bd6c533-d592-4bb1-a644-83a4befb1f7a" />

It then sets it when this path is "" here as a default. -> pkg/cli/alpha/internal/generate.go:246

At least this is how I understand it and I get a `output-dir` folder.

Reply:
OH we need to fix that.
It is a bug. By default it should be current dir == PWD

Reply:
Just to get this right: So instead of appending `output-dir` to the PWD it should be just the PWD right? 

I can make a PR for this & maybe this then also removes the need to change the docs.

Reply:
So something like this? 

<img width="1177" alt="image" src="https://github.com/user-attachments/assets/c4d9fe89-2bd8-454d-96fb-e51bf352b87c" />

Reply:
We need to do a little more than that.
Also, we need to ensure it via the tests.
See: https://github.com/kubernetes-sigs/kubebuilder/pull/4500